src/libyaml/patches/CorrectWarningsAndBuildIssuesInLibYaml.patch

```diff
@@ -0,0 +1,131 @@
+#Date: May 10, 2024
+#Subject: [PATCH] Patch to fix compilation warning
+#		 in libfyaml-master directory. 
+#
+#Source: Uses code from the libfyaml-master and asprintf.c 
```

Comment:
Please add an extra line in the header to say:
# See NOTICE for license details.
(just to make the licensing of this patch a little clearer)